site/class-1-bar/LeonvanZijl/LeonvanZijl/data.tsv

```diff
@@ -0,0 +1,8 @@
+letter		frequency
```

Comment:
This also looks great, but you’re also uploading files a directory to deep: `site/class-1-bar/LeonvanZijl/LeonvanZijl/` instead of `site/class-1-bar/LeonvanZijl/`.